src/librustdoc/markdown.rs

```diff
@@ -35,14 +35,13 @@ fn extract_leading_metadata(s: &str) -> (Vec<&str>, &str) {
 /// Render `input` (e.g., "foo.md") into an HTML file in `output`
 /// (e.g., output = "bar" => "bar/foo.html").
 pub fn render(
-    input: PathBuf,
+    input: &PathBuf,
```

Comment:
Should this not be a `&Path` or even an `AsRef<Path>` to be more idiomatic?

Reply:
Absolutely!